lib/xboxrt/vcruntime/vcruntime_typeinfo.h

```diff
@@ -25,12 +25,13 @@ extern "C"
 class type_info
 {
 public:
-    virtual ~type_info ();
+    virtual ~type_info () noexcept;
 
     const char *name () const noexcept
     {
         // FIXME: return human-readable name
-        assert(false);
+        //assert(false);
+        return this->_Data._UndecoratedName;
         return NULL;
```

Comment:
Ah! My famous `return; return` performance optimization. Returns 1 instruction earlier than it would normally do.

Reply:
Yeah that's just wip hackery.